### PR TITLE
German Luppp translation

### PR DESCRIPTION
**Pull request by coderkun**
Translate the Luppp manual into German using the file “de-luppp.rst”. Please read and review.

### CODE REVIEW

**Comment by coderkun** [comment]
Not complete, yet, so please wait with merging.

**Comment by coderkun** [comment]
The Luppp translation is complete now. @georgkrause, please review. Feel free to squash the commits.

**Comment by georgkrause** [comment]
@coderkun Thanks for Your work. Sadly i am I'll and away from my devices so this will take some time. I hope I can do it next week.

**Comment by georgkrause** [comment]
*ill. Funny Auto correction :D

**Comment by georgkrause** [comment]
@coderkun i would suggest to change this in order to fit your issues. eg replace all "we"s with "you". Or do you think its better to keep it with the original english version?

File: source/de-luppp.rst [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+.. image:: img/luppp/luppp.png
+   :align: center
+
+Luppp ist ein Programm zur Musikerstellung – vorallem für den Live-Einsatz. Der
+Fokus liegt auf Echtzeitverarbeitung und intuitivem Workflow. Durch
+umfangreiche MIDI-Steuerung wird der Einsatz von Loops zum Kinderspiel! In den
+nächsten Abschnitten wird beschrieben, wie Luppp zusammen mit JACK für
+Audioein- und -ausgabe eingerichtet wird. Außerdem werden die Konzepte von
+Szenen und Clips vorgestellt, sodass die Kreativität für Live-Auftritte optimal
```
Comment (georgkrause) [inline source/de-luppp.rst]: "sodass du deiner Kreativität freien Lauf lassen kannst" ?

File: source/de-luppp.rst [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+genutzt werden kann.
+
+Luppp und JACK
+==============
+
+Wenn dir Luppp und Jack komplett neu sind, schaue dir dieses Beispielvideo an,
+das eine typische Looping-Session mit Luppp zeigt:
+https://www.youtube.com/watch?v=R6WiWDDKRCQ
+
+Bevor wir Loops aufnehmen können, müssen wir eine Audioquelle mit Luppps
```
Comment (georgkrause) [inline source/de-luppp.rst]: "Bevor du Loops aufnehmen kannst"

File: source/de-luppp.rst [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+“master_in”-Anschluss verbinden. Außerdem müssen “master_right” und
+“master_left” mit den Systemanschlüssen (Soundkarte oder Audiointerface)
+verbunden werden. Der folgende Screenshot zeigt die Verbindungen:
+
+.. image:: img/luppp/luppp_jack_connections.png
+   :align: center
+
+Als nächstes schauen wir uns die Oberfläche von Luppp an. Diese mag auf den
```
Comment (georgkrause) [inline source/de-luppp.rst]: "Schau dir nun die Oberfläche von Luppp an"

File: source/de-luppp.rst [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+ersten Blick komplex wirken, ist aber wirklich einfach, wenn man die einzelnen
+Bereiche ersteinmal versteht.
+
+.. image:: img/luppp/luppp_interface.png
+   :align: left
+
+Du siehst acht Tracks mit jeweils zehn Clips. Um zu testen, ob dein Audiosetup
+funktioniert, spiele deine Audioquelle ab und du solltest die Amplitude auf der
```
Comment (georgkrause) [inline source/de-luppp.rst]: statt Amplitude eventuell Ausschlag? oder Pegel? Das klingt weniger technisch

File: source/de-luppp.rst [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+Standardwert ist 120).
+
+.. image:: img/luppp/luppp_scenes.png
+	:align: right
+
+Szenen
+======
+
+Ein sehr wichtiges Konzezt von Luppp sind die Szenen. Auf jedem Track kann nur
+ein Clip zur gleichen Zeit agbespielt werden. Im Masterbereich von Luppp
```
Comment (georgkrause) [inline source/de-luppp.rst]: "abgespielt" ;)

**Comment by coderkun** [comment]
@georgkrause, I would like to do this as separate steps:

1. Merge the current German translation
2. Fix the wording in the English version (if @harryhaaren agrees to this change, it is currently only a suggestion)
3. Update the German translation to reflect the English wording

File: source/de-luppp.rst [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+-----
+
+Panning
+=======
+
+Bevor wir uns der Welt der Bearbeitung in Lupp zuwenden, schauen wir uns eine
+einfache, aber wirkungsvolle Funktion an. Es wurde in Version 1.1.1 eingeführt:
+das Panning. Der Regler über dem Fader jeden Tracks kann das Signal zum
+Masterkanal splitten. Du hast also die Möglichkeit, jedem Kanal einen Platz im
+Mix zu geben. Das ist sehr simple, gehen wir also weiter …
```
Comment (georgkrause) [inline source/de-luppp.rst]: "simpel"

File: source/de-luppp.rst [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+
+FX-Bereich
+==========
+
+Die Regler direkt unter dem Clip-Bereich sind der Effektbereich. In dem
+JACK-Verbindungsmanager deiner Wahl werden dir die „Send_track_X“- und
+„Return_track_X“-Anschlüsse aufgefallen sein. Die Verbindung des
+„send“-Anschlusses mit dem Eingang eines Plugins und des Ausganges mit dem
+„return“-Anschluss von Luppp leitet das Audiosignal durch dieses Plugin. Um die
+Bearbeitung zu aktivieren, klicke auf die gelbe FX-Schaltfläche. Mit dem
```
Comment (georgkrause) [inline source/de-luppp.rst]: "Bearbeitung" erscheint mir hier nicht als das richtige Wort, eine bessere Idee habe ich aber leider auch nicht :(

**Comment by harryhaaren** [comment]
Yep whatever suits you guys best - both OK with me!

File: source/de-luppp.rst [hunk 8]
```diff
@@ -0,0 +1,280 @@
...
+Der Standardwert ist ``SINC_FASTEST`` (``1``)::
+
+    "resampleQuality": 1,
+
+Standard-Controller
+-------------------
+
+Mit der Option ``defaultControllers`` können Controller definiert werden, die
+beim Programmstart automatisch geladen werden. Mehrere Controller können mit
+Komma getrennt ngegeben werden::
```
Comment (georgkrause) [inline source/de-luppp.rst]: "angegeben"

**Review by georgkrause — CHANGES_REQUESTED** [review]
I added some comments what could be changed, please review!
Why are there changes in the luppp.rst? This shouldnt be part of this PR

**Comment by georgkrause** [comment]
@coderkun i am okay with your workflow, so lets keep it according to the english wording.

File: source/de-luppp.rst (hunk 1 above)
Comment (coderkun) [reply source/de-luppp.rst]: Much better, yes.

File: source/de-luppp.rst (hunk 2 above)
Comment (coderkun) [reply source/de-luppp.rst]: Sounds better, I agree.

File: source/de-luppp.rst (hunk 3 above)
Comment (coderkun) [reply source/de-luppp.rst]: Agreed.

File: source/de-luppp.rst (hunk 4 above)
Comment (coderkun) [reply source/de-luppp.rst]: In diesem Fall finde ich „Ausschlag“ am verständlichsten.

File: source/de-luppp.rst (hunk 5 above)
Comment (coderkun) [reply source/de-luppp.rst]: Typo, right.

File: source/de-luppp.rst (hunk 6 above)
Comment (coderkun) [reply source/de-luppp.rst]: Typo, right.

File: source/de-luppp.rst (hunk 7 above)
Comment (coderkun) [reply source/de-luppp.rst]: Vielleicht einfach „Um den Effekt zu nutzen, klick auf […]“?

File: source/de-luppp.rst (hunk 8 above)
Comment (coderkun) [reply source/de-luppp.rst]: Typo, right.

File: source/de-luppp.rst (hunk 7 above)
Comment (georgkrause) [reply source/de-luppp.rst]: klingt gut.

**Review by coderkun — COMMENTED** [review]
Requested changes have been pushed with commit 6aee873.

**Comment by coderkun** [comment]
@georgkrause, is this PR complete or is anything missing?

**Comment by georgkrause** [comment]
@coderkun oh sorry, i totally missed this! i will look about it now

**Review by georgkrause — APPROVED** [review]

**Pull request merged**